pkg/apis/workflow/v1alpha1/workflow_types.go

```diff
@@ -434,7 +434,7 @@ func (s ShutdownStrategy) ShouldExecute(isOnExitPod bool) bool {
 
 // +kubebuilder:validation:Type=array
 type ParallelSteps struct {
-	Steps []WorkflowStep `json:",inline" protobuf:"bytes,1,rep,name=steps"`
```

Comment:
Is this intended? I'm not certain, but this could be a big change?

Reply:
this recent change (2-3 day old) change needed to be reverted because it does not work with the new code